lmod_proxy/edx_grades/__init__.py

```diff
@@ -14,13 +24,51 @@
 )
 
 
-@edx_grades.route('/', methods=['GET'])
+@edx_grades.route('', methods=['GET', 'POST'])
 def index():
     """Handle ``POST`` from edx-platform or print available actions, and
     provide an interactive test of the available actions on ``GET``.
 
     Returns:
-        flask.response
-
+        flask.response: JSON on POST, HTML on GET
     """
-    return '<h1>LMod Proxy edX Grades "API"</h1>'
+    requestor = request.remote_addr
+    if request.headers.getlist("X-Forwarded-For"):
+        requestor = request.headers.getlist("X-Forwarded-For")[0]
+
+    if request.method == 'POST':
+        form = EdXGradesForm(request.form)
```

Comment:
grades2stellar had a simple form for exercising calls to the Learning Modules web service.  This form performs the same function by calling the PyLmod API and returning the results.